### ISSUE TITLE
On Java 11 throws a org.springframework.web.client.HttpClientErrorException$NotFound

### ISSUE THREAD

**Issue by LProemer** [porscheinformatik/zanata-spring#30]
Hi: 
On Java 10, spring 5
Trying to load a non existent Translation throws a org.springframework.web.client.HttpClientErrorException
could this be catched please?

Lg Lukas

**Comment by derkoe** [comment]
First, do not use Java 10! Either Java 8 or 11 (or soon 17).

Second, please provide some more error details.

**Comment by LProemer** [comment]
My bad. We do use Java 11.
This is the Stacktrace:
`org.springframework.web.client.HttpClientErrorException$NotFound: 404 Not Found: [no body]
	at org.springframework.web.client.HttpClientErrorException.create(HttpClientErrorException.java:113)
	at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:170)
	at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:112)
	at org.springframework.web.client.ResponseErrorHandler.handleError(ResponseErrorHandler.java:63)
	at org.springframework.web.client.RestTemplate.handleResponse(RestTemplate.java:785)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:743)
	at org.springframework.web.client.RestTemplate.exchange(RestTemplate.java:644)
	at at.porscheinformatik.zanata.ZanataMessageSource.loadTranslation(ZanataMessageSource.java:232)
	at at.porscheinformatik.zanata.ZanataMessageSource.loadTranslations(ZanataMessageSource.java:194)
	at at.porscheinformatik.zanata.ZanataMessageSource.resolveCodeTextFlowTarget(ZanataMessageSource.java:278)
	at at.porscheinformatik.zanata.ZanataMessageSource.resolveCodeWithoutArguments(ZanataMessageSource.java:272)`

Basically when a resource is not found, instead of silently using the fallback and producing a warning in the log this error is thrown.

**Comment by derkoe** [comment]
The error is caught and logged with level "WARN" (like in the provided stacktrace) as expected.
